src/mame/drivers/seta.cpp

```diff
@@ -12324,6 +12362,7 @@ GAME( 1993, atehate,   0,        atehate,   atehate,   seta_state,     empty_ini
 GAME( 1993, daioh,     0,        daioh,     daioh,     seta_state,     empty_init,     ROT270, "Athena",                    "Daioh", 0 )
 GAME( 1993, daioha,    daioh,    daioh,     daioh,     seta_state,     empty_init,     ROT270, "Athena",                    "Daioh (earlier)", 0 )
 GAME( 1993, daiohp,    daioh,    daiohp,    daiohp,    seta_state,     empty_init,     ROT270, "Athena",                    "Daioh (prototype)", 0 )
+GAME( 1993, daiohp2,   0,        daiohp,    daiohp,    seta_state,     empty_init,     ROT270, "Athena",                    "Daioh (prototype, earlier)", 0 )
```

Comment:
Shouldn’t it be a clone of daioh?

Reply:
In the future, try to be more helpful with your comments. I had to go and visually compare a bunch of hashes before realizing that "gfx2" and "gfx3" were the same between daiohp2 and daiohp, which you could have made clearer in the first place.

Reply:
Yes it's a MAME clone set of daioh, no complicated reasons for it.